sdk/healthinsights/azure-healthinsights-radiologyinsights/README.md

```diff
@@ -0,0 +1,146 @@
+| 1.0.0b1     | 2023-09-01-preview               |
+
+
+### Authenticate the client
+
+#### Get the endpoint
+
+You can find the endpoint for your Health Insights service resource using the [Azure Portal][azure_portal] or [Azure CLI][azure_cli]
+
+
+```bash
+# Get the endpoint for the Health Insights service resource
+az cognitiveservices account show --name "resource-name" --resource-group "resource-group-name" --query "properties.endpoint"
+```
+
+#### Get the API Key
+
+You can get the **API Key** from the Health Insights service resource in the Azure Portal.
+Alternatively, you can use **Azure CLI** snippet below to get the API key of your resource.
+
+```PowerShell
+az cognitiveservices account keys list --resource-group <your-resource-group-name> --name <your-resource-name>
+```
+
+#### Create a RadiologyInsightsClient with an API Key Credential
+
+Once you have the value for the API key, you can pass it as a string into an instance of **AzureKeyCredential**. Use the key as the credential parameter to authenticate the client:
+
+```python 
+\<!-- SNIPPET:sample_critical_result_inference_async.create_radiology_insights_client-->
+import os
+from azure.core.credentials import AzureKeyCredential
+from azure.healthinsights.clinicalmatching import ClinicalMatchingClient
+
+KEY = os.environ["AZURE_HEALTH_INSIGHTS_API_KEY"]
+ENDPOINT = os.environ["AZURE_HEALTH_INSIGHTS_ENDPOINT"]
+
+radiology_insights_client = RadiologyInsightsClient(endpoint=ENDPOINT, credential=AzureKeyCredential(KEY))
+\<!-- SNIPPET:sample_critical_result_inference_async.create_radiology_insights_client-->
+```
+
+### Long-Running Operations
+
+Long-running operations are operations which consist of an initial request sent to the service to start an operation,
+followed by polling the service at intervals to determine whether the operation has completed or failed, and if it has
+succeeded, to get the result.
+
+Methods that support healthcare analysis, custom text analysis, or multiple analyses are modeled as long-running operations.
+The client exposes a `begin_<method-name>` method that returns a poller object. Callers should wait
+for the operation to complete by calling `result()` on the poller object returned from the `begin_<method-name>` method.
+Sample code snippets are provided to illustrate using long-running operations [below](#examples "Examples").
+
+## Key concepts
+
+Radiology Insights currently supports one document from one patient. Please take a look [here][inferences] for more detailed information about the inferences this service produces. 
+
+## Examples
+
+### Create a RadiologyInsights request and get the result using an asynchronous client
+
+For an example how to create a client, a request and get the result see the example in the sample folder.
+
+### Get Critical Result Inference information
+
+```python
+\<!-- SNIPPET:sample_critical_result_inference_async.display_critical_results-->
+def display_critical_results(self,radiology_insights_result):
```

Comment:
I would expand this snippet to include the method call or if you're only interested in showing how to access result data, then probably remove this line and just focus on the result handling.

Reply:
took the def out as i really just wanted to show how to access the result